stingray/tests/test_crossspectrum.py

```diff
@@ -322,19 +322,21 @@ def test_different_dt(self):
 
     def test_different_tseg(self):
         time2 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
-        counts2_test = np.random.poisson(0.01, size=len(time2))
+        counts2_test = np.random.poisson(1000, size=len(time2))
         test_lc2 = Lightcurve(time2, counts2_test)
 
         time1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-        counts1_test = np.random.negative_binomial(1, 0.09, size=len(time1))
+        counts1_test = np.random.np.random.poisson(1000, size=len(time1))
```

Comment:
Here I'm wondering about the other one: why did we choose a negative binomial here???

Reply:
No idea :D